databrowser/src/config/tourism/eventTopics/eventTopics.operations.ts

```diff
@@ -0,0 +1,34 @@
+import { Operations } from '../../../domain/datasetConfig/types';
+import {
+  extendCreateRoles,
+  extendDeleteRoles,
+  extendUpdateRoles,
+  ROLE_READ,
+} from '../roles';
+
+export const eventTopicsOperations: Operations = {
+  readAll: {
+    rolesAllowed: ROLE_READ,
+  },
+  read: {
+    rolesAllowed: ROLE_READ,
+  },
+  create: {
+    rolesAllowed: extendCreateRoles([
+      'eventTopicsManager',
+      'eventTopicsCreate',
```

Comment:
Roles are case-sensitive, in this case the correct role names are "EventTopicsManager" and "EventTopicsCreate".